lab_1/README.md

```diff
@@ -1,6 +1,6 @@
 # Let's start with the dot
 
-Start by opening three tabs in our browser:
+Start by opening three tabs in ypur browser:
```

Comment:
not sure about this one, it's technically correct. If you accept the change, please note I made a typo - change it to 'your'